ably-iosTests/ARTRestChannelPublishTest.m

```diff
@@ -98,26 +97,6 @@ -(void) testPublishArray {
     [self waitForExpectationsWithTimeout:[ARTTestUtil timeout] handler:nil];
 }
 
-
-/**
- Currently the payloadArraySizeLimit is default to INT_MAX. Here we bring that number down to 2
- To show that publishing an array over the limit throws an exception.
- */
--(void) testPublishTooManyInArray {
-    XCTestExpectation *exp = [self expectationWithDescription:@"testPublishTooManyInArray"];
-    [ARTTestUtil testRest:^(ARTRest *rest) {
-        _rest = rest;
-        ARTChannel *channel = [rest.channels get:@"channel"];
-        NSArray *messages = @[[[ARTMessage alloc] initWithData:@"test1" name:nil],
-                              [[ARTMessage alloc] initWithData:@"test2" name:nil],
-                              [[ARTMessage alloc] initWithData:@"test3" name:nil]];
-        [ARTPayload getPayloadArraySizeLimit:2 modify:true];
```

Comment:
The spec says: "However, a yet to be implemented feature should limit the total number of messages bundled in a single ProtocolMessage based on the default max message size". What was in there was implementing something that isn't specified, and thus has no API defined, and furthermore, applied a limit on the number of messages when the spec says the limit would be based on "the default max message size".